loader/proto.go

```diff
@@ -207,12 +207,14 @@ func (b *builder) handleMessageField(w *strings.Builder, field proto.Visitee) er
 		name = field.Name
 		typ = b.goType(field.Type)
 		sequence = field.Sequence
+		b.updateComment(field.Comment, name)
 		comment = field.Comment
 		repeated = field.Repeated
 		options = field.Options
 	case *proto.MapField:
 		name = field.Field.Name
 		sequence = field.Field.Sequence
+		b.updateComment(field.Comment, name)
```

Comment:
can you not move this line to right after the switch case, and deduplicate it?

in fact, that would probably mean you no longer need the method.